deps/make.blosc

```diff
@@ -89,3 +91,16 @@ $(LZ4_DIR)/lz4.o: $(LZ4_DIR)/lz4.c
 
 $(LZ4_DIR)/lz4hc.o: $(LZ4_DIR)/lz4hc.c
 	$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $(LZ4_DIR)/lz4hc.c
+
+$(SNAPPY_DIR)/snappy-c.o: $(SNAPPY_DIR)/snappy-c.cc
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) -c -o $@ $(SNAPPY_DIR)/snappy-c.cc
```

Comment:
You also need to use `$(CXX)` and `$(CXXFLAGS)` for linking with the `-shared` flag, to make sure it links the C++ standard library.  Otherwise cross-compiling fails (I just tried it).

(The C++ standard library worries me a bit...hopefully we will not get library conflicts when we try to load it if the precompiled Mac or Windows binaries were linked with a different version.)